crates/teloxide-core/src/types/reply_keyboard_markup.rs

```diff
@@ -29,14 +29,16 @@ pub struct KeyboardMarkup {
     /// (e.g., make the keyboard smaller if there are just two rows of
     /// buttons). Defaults to `false`, in which case the custom keyboard is
     /// always of the same height as the app's standard keyboard.
-    pub resize_keyboard: Option<bool>,
+    #[serde(skip_serializing_if = "std::ops::Not::not")]
+    pub resize_keyboard: bool,
 
     /// Requests clients to hide the keyboard as soon as it's been used. The
     /// keyboard will still be available, but clients will automatically
     /// display the usual letter-keyboard in the chat – the user can press a
     /// special button in the input field to see the custom keyboard again.
     /// Defaults to `false`.
-    pub one_time_keyboard: Option<bool>,
+    #[serde(skip_serializing_if = "std::ops::Not::not")]
+    pub one_time_keyboard: bool,
 
     /// The placeholder to be shown in the input field when the keyboard is
     /// active; 1-64 characters.
```

Comment:
Funnily enough, we could make `input_field_placeholder` into just a `String` too, since it requires at least one character:
```rust
#[serde(skip_serializing_if = "str::is_empty")]
pub input_field_placeholder: String,
```
But I'm not sure if it's any better...

Reply:
Well, at least I don't see why we need `Option` here, so I'm inclining towards `String`.